src/Main.java

```diff
@@ -61,6 +69,8 @@ public static void startAndWaitForThreads(Runnable r, int num) {
                 e.printStackTrace();
             }
         }
+        long dur = System.currentTimeMillis() - start;
+        return dur;
```

Comment:
Good stuff - I should probably change runIterations() in Benchmarking so that it makes use of this instead